card/lib/cardio.rb

```diff
@@ -19,9 +19,32 @@ module Cardio
   extend Delaying
   CARD_GEM_ROOT = File.expand_path("..", __dir__)
 
-  mattr_reader :paths, :config
+  module RailsConfigMethods
```

Comment:
modules like this should be in cardio subdir.  (eg cardio/rails_config_methods.rb). Those will need to be explicitly required.